woodwork/serialize.py

```diff
@@ -178,6 +178,12 @@ def write_dataframe(dataframe, path, format='csv', **kwargs):
         dataframe[latlong_columns] = dataframe[latlong_columns].astype(str)
 
         dataframe.to_parquet(file, **kwargs)
+    elif format == 'arrow':
+        dataframe = dataframe.ww.copy()
+        latlong_columns = [col_name for col_name, col in dataframe.ww.columns.items() if _get_ltype_class(col.logical_type) == ww.logical_types.LatLong]
+        dataframe[latlong_columns] = dataframe[latlong_columns].astype(str)
```

Comment:
Seems like this has the same issue as with `parquet` where it can't handle tuples. Can we reuse the code from above somehow?

Reply:
yep!